websites/visualization/src/app/react/experimental/wrap/after/page.tsx

```diff
@@ -0,0 +1,23 @@
+'use client'
+import { ErrorBoundary, ErrorBoundaryGroup, Suspense, useErrorBoundary, wrap } from '@suspensive/react'
+import { UseSuspenseQuery } from '~/components'
+import { api } from '~/utils'
+
+const logError = (error: Error) => console.error(error)
+
+export default wrap(ErrorBoundaryGroup, { blockOutside: false })(
+  wrap(ErrorBoundary, { fallback: (props) => <>{props.error.message}</>, onError: logError })(
+    wrap(Suspense.CSROnly, { fallback: <>loading...</> })(() => {
+      const errorBoundary = useErrorBoundary()
+
+      return (
+        <>
+          <button onClick={() => errorBoundary.setError(new Error('trigger error by useErrorBoundary().setError'))}>
+            trigger error by useErrorBoundary().setError
+          </button>
+          <UseSuspenseQuery queryKey={['with', 1] as const} queryFn={() => api.delay(200, { percentage: 50 })} />
+        </>
+      )
+    })
+  )
+)
```

Comment:
We can compare before, after of `wrap` function 
between `websites/visualization/src/app/react/experimental/wrap/after/page.tsx` and `websites/visualization/src/app/react/experimental/wrap/before/page.tsx`

Check it please

Reply:
<img width="1221" alt="image" src="https://github.com/suspensive/react/assets/57122180/477676aa-47ab-40ed-92fc-ef6ad4b7130e">


<img width="1257" alt="image" src="https://github.com/suspensive/react/assets/57122180/5810c170-70ac-4c09-a4ef-542c8619fd3f">


Can I compare it through the network tab? Since the user doesn't write the `wrap` function himself, do I just need to look at the performance aspect?

Reply:
> Can I compare it through the network tab? Since the user doesn't write the wrap function himself, do I just need to look at the performance aspect?
Sure, Could you please suggest how can we compare performance?

however, this change can greatly reduce the file size of the library without BREAKING CHANGE. This is a definite improvement. Additionally, the number of public APIs users have to know can be greatly reduced. Although I think this is a more important improvement than performance. So I hope that the performance comparison does not delay the merge of this feature. Before merging, it would be better to talk about the interface of `wrap (tentative name)` rather than comparing performance.

Reply:
Oh I see, I agree about your opinion